netbox/dcim/filtersets.py

```diff
@@ -1725,6 +1727,18 @@ def filter_by_termination(self, queryset, name, value):
         # Supported objects: device, rack, location, site
         return queryset.filter(**{f'terminations___{name}__in': value}).distinct()
 
+    def filter_by_cable_end(self, queryset, name, value, side):
+        # Filter by termination id and cable_end type
+        return queryset.filter(**{f'{name}__in': value, 'terminations__cable_end': side}).distinct()
+
+    def filter_by_cable_end_a(self, queryset, name, value):
```

Comment:
Rather than introducing two standalone methods here, we could derive the A/B side from the filter name (the `name` argument), and have both filters above call `filter_by_cable_end()` directly.

Reply:
name is the field name, which is the same in both cases "terminations__termination_id"